main/guides/integration/chain-integration.md

```diff
@@ -25,14 +20,6 @@ The chain can also be accessed via JavaScript using the [`cosmjs` library](https
 - The base unit for staking is `ubld` (corresponding to `uatom` for Cosmos Hub)
 - The command utility of the agoric chain is `agd` (corresponding to [`simd` for the Cosmos Hub](https://docs.cosmos.network/v0.45/run-node/interact-node.html)). 
 ---
-# Tools
-## Building `agd`
-
-The `agd` command line tool can be built as described in the Agoric [getting-started documentation](https://docs.agoric.com/guides/getting-started#build-the-cosmic-swingset-package). The linked step builds `agd`. To confirm that `agd` is in your `$PATH`, execute
-```
-agd version --long
-```
-
```

Comment:
Rather than deleting this section, how about adapting info from the **Installing agd** box in https://docs.agoric.com/guides/agoric-cli/agd-query-tx.html ?

Reply:
The info there does not say much about **building agd** though - it is about how to use an already built `agd` in docker container. Should I change the heading of the subsection to **Installing agd** too?

Reply:
it has 2 options: a docker container or building from a release.

Maybe the thing to do is to keep this section as "Building agd" and make it mostly a pointer to https://github.com/Agoric/agoric-sdk/releases ; the docker option is probably less relevant here.